src/boundaries/edgetracking.jl

```diff
@@ -1,11 +1,31 @@
-export edgetracking, bisect_to_edge
+export edgetracking, bisect_to_edge, EdgeTrackingResults
 
 """
-    edgetracking(ds::DynamicalSystem, attractors::Dict; kwargs...) -> edge, track1, track2
+    EdgeTrackingResults(edge, track1, track2, time, bisect_idx)
+Data type that stores output of the [`edgetracking`](@ref) algorithm.
+
+## Fields
+* `edge::StateSpaceSet`: the pseudo-trajectory representing the edge track
+* `track1::StateSpaceSet`: the parallel pseudo-trajectory tracking the edge within basin 1
+* `track2::StateSpaceSet`: the parallel pseudo-trajectory tracking the edge within basin 2
+* `time::Vector`: time points of the above `StateSpaceSet`s
+* `bisect_idx::Vector`: indices of `time` at which a re-bisection was done
+"""
+struct EdgeTrackingResults
+    edge::StateSpaceSet
+    track1::StateSpaceSet
+    track2::StateSpaceSet
+    time::Vector{Float64}
+    bisect_idx::Vector{Int}
+end
+
+"""
+    edgetracking(ds::DynamicalSystem, attractors::Dict; kwargs...)
 Track along a basin boundary in a dynamical system `ds` with two or more `attractors`
 in order to find an *edge state* or saddle.
-Return a pseudo-trajectory `edge` describing the boundary, derived from two shadowing
-trajectories `track1`, `track2` tracking along either side of the boundary.
+Return a pseudo-trajectory `edge` describing the boundary, derived from two
```

Comment:
You need to say something like "return the results in the form of [`EdgetrackingR...`] which contains the tracked edge state".